Classes/BEMCheckBox.m

```diff
@@ -172,6 +172,27 @@ - (void)handleTapCheckBox:(UITapGestureRecognizer *)recognizer {
     }
 }
 
+// increase touch area
```

Comment:
Please remove this comment add a pragrma mark instead.  
Also, the method should be moved under the `Helper Methods` section.